docker-images/zrok-private-share/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3'
+services:
+  zrok-enable-init:
+    image: busybox
+    # matches uid:gid of "nobody" in zrok container image
+    command: chown -Rc 65534:65534 /mnt/.zrok
+    user: root
+    volumes:
+      - zrok_env:/mnt/.zrok
+  zrok-enable:
+    image: docker.io/openziti/zrok:enable-headless  # FIXME: resume :latest if >= :0.3.2
+    depends_on:
+      zrok-enable-init:
+        condition: service_completed_successfully
+    entrypoint:
+      - bash
+      - -c
+      - |
+        if [[ -s /mnt/.zrok/environment.json ]]; then
+          echo "INFO: noop: zrok environment is already enabled"
+          exit 0
+        else
+          echo "INFO: running: zrok $$(sed -E 's/\S{12,}/************/' <<< $${@})"
```

Comment:
just noticed that the doubled $$ are in these files

Reply:
they're needed to escape vars in Compose syntax